judge/views/submission.py

```diff
@@ -393,9 +393,11 @@ def get_title(self):
 
     def get_content_title(self):
         if self.is_own:
-            return format_html('All my submissions')
-        return format_html('All submissions by <a href="{1}">{0}</a>', self.profile.display_name,
-                           reverse('user_page', args=[self.username]))
+            return mark_safe(escape(_('All my submissions')))
```

Comment:
```suggestion
            return _('All my submissions')
```

This is sufficient, no need for cargo-culting the `format_html` logic.